images/ceph/toolbox.sh

```diff
@@ -22,19 +22,19 @@ KEYRING_FILE="/etc/ceph/keyring"
 # without specifying any arguments
 write_endpoints() {
   endpoints=$(cat ${MON_CONFIG})
-  
+
   # filter out the mon names
   # external cluster can have numbers or hyphens in mon names, handling them in regex
   # shellcheck disable=SC2001
   mon_endpoints=$(echo "${endpoints}"| sed 's/[a-z0-9_-]\+=//g')
-  
+
   DATE=$(date)
   echo "$DATE writing mon endpoints to ${CEPH_CONFIG}: ${endpoints}"
     cat <<EOF > ${CEPH_CONFIG}
 [global]
 mon_host = ${mon_endpoints}
 
-[client.admin]
+[client.rookoperator]
```

Comment:
The toolbox should use a `rooktoolbox` username instead of `rookoperator`. But maybe don't worry about that until you get the main scenarios working first with this.